### PR TITLE
Document the .value() and .gas() modifiers

### LINKED ISSUE #1363: Address methods are not properly documented
Addresses seem to have the following methods:

balance()
send()
call()
callcode()
delegatecall()

Some of these methods can be chained with gas() and value()

I can't find any good documentation on these methods.

Currently there is:
http://solidity.readthedocs.io/en/develop/types.html#address
http://solidity.readthedocs.io/en/develop/units-and-global-variables.html#address-related

### PR DESCRIPTION
**Pull request by axic**
Fixes #1363.

### CODE REVIEW

File: docs/control-structures.rst [hunk 1]
```diff
@@ -113,8 +113,8 @@ actual contract has not been created yet.
 Functions of other contracts have to be called externally. For an external call,
 all function arguments have to be copied to memory.
 
-When calling functions
-of other contracts, the amount of Wei sent with the call and the gas can be specified::
+When calling functions of other contracts, the amount of Wei sent with the call and
+the gas can be specified with special modifiers ``.value()`` and ``.gas()``, respectively::
```
Comment (chriseth) [inline docs/control-structures.rst]: I think we should also mention that `value` is only available for `payable` functions.

**Review by chriseth — CHANGES_REQUESTED** [review]
It could be quite confusing to call these `modifiers`. Do they need a name?

**Comment by axic** [comment]
> It could be quite confusing to call these `modifiers`. Do they need a name?

They are quite a strange language feature though, a kind of modifier on the function type. Any wording suggestion?

**Comment by chriseth** [comment]
As I said, I think we can work around by not giving them a name: "`.gas()` and `.value()` are only available..."
We can call them by there more generic term "members".

File: docs/control-structures.rst (hunk 1 above)
Comment (axic) [reply docs/control-structures.rst]: Reworded it.

**Comment by axic** [comment]
How about "with special settings `.value()` and `.gas()`"? Or "options".

File: docs/control-structures.rst (hunk 1 above)
Comment (chriseth) [reply docs/control-structures.rst]: `specified with the special functions ``.value()``...`?

File: docs/control-structures.rst [hunk 2]
```diff
@@ -127,8 +127,8 @@ of other contracts, the amount of Wei sent with the call and the gas can be spec
         function callFeed() { feed.info.value(10).gas(800)(); }
     }
 
-The modifier ``payable`` has to be used for ``info``, because otherwise,
-we would not be able to send Ether to it in the call ``feed.info.value(10).gas(800)()``.
+The modifier ``payable`` has to be used for ``info``, because otherwise, the `.value()`
+modifier would not be available.
```
Comment (chriseth) [inline docs/control-structures.rst]: `because otherwise, `.value()` would not be available.`

**Comment by axic** [comment]
Changed to option, felt way more natural.

**Review by chriseth — APPROVED** [review]

**Pull request merged**